src/app/integrations/mailchimp/client.py

```diff
@@ -40,7 +40,7 @@ def mass_subscribe(self, list_id: str, members: Iterable[MailchimpMember]):
                 'update_existing': True,
             },
         )
-        if len(response['errors']):
+        if response['errors']:
```

Comment:
Unfortunately this change brakes [this test](https://github.com/f213/education-backend/blob/master/src/app/tests/mailchimp/tests_mailchimp_mass_subscribe.py#L13). You may check it locally by running `make test`.
```suggestion
        if 'errors' in response and len(response['errors']):
```